t/plugin/authz-keycloak.t

```diff
@@ -314,7 +314,7 @@ GET /t
 --- response_body
 false
 --- error_log
-Error while sending authz request to https://127.0.0.1:8443/auth/realms/University/protocol/openid-connect/token: 18: self signed certificate
+Error while sending authz request to https://127.0.0.1:8443/auth/realms/University/protocol/openid-connect/token: 18
```

Comment:
why change this?

Reply:
Openssl 3.0 changes the error log content, so we must adapt it.

Reply:
@moonming I already manged to run and pass all tests related to SSL. But according to @membphis , I need to limit the test cases.

Reply:
> 

What is the error message of OpenSSL 3.0?

Reply:
In most cases, self assgined cert reject would be "18: self-signed certificate"

Reply:
Understood. So we can be compatible with it not removed it.

Reply:
agree with @moonming